welly/las.py

```diff
@@ -0,0 +1,272 @@
+import lasio
+import numpy as np
+import pandas as pd
+from lasio import HeaderItem, CurveItem, SectionItems
+import warnings
+
+from welly.fields import curve_sections, other_sections, header_sections
+from welly.utils import get_columns_decimal_formatter
+
+# get header item dictionary
+header_item = HeaderItem().__dict__
+
+# get curve item dictionary
+curve_item = CurveItem().__dict__
+
+pd.set_option('precision', 10)
+
+
+def from_las(path, **kwargs):
+    """
+    Read las file and parse every dataset to 2 pd.DataFrames:
+        1. curve data
+        2. header metadata
+
+    Only LAS 1.2 and 2.0 are currently supported. LAS 3.0 will be supported when lasio LAS 3.0 work is finished:
+    https://github.com/kinverarity1/lasio/issues/5. The design of this reader already accommodates for LAS 3.0
+    functionality where 1 lasio.LASFile can contain multiple 1D, 2D or 3D dataset entries.
+
+    Args:
+        path (str): Path to LAS file
+        **kwargs: The additional keyword arguments are propagated to the lasio reader so you can use when reading in a
+                  LAS file. Find the routines of the different keyword possibilities look here:
+                    * :func:`lasio.reader.open_with_codecs` - manage issues relate to character encodings
+                    * :meth:`lasio.LASFile.read` - control how NULL values and errors are handled during parsing
+
+    Returns:
+        datasets (Dict['dataset_name': (data (pd.DataFrame), header (pd.DataFrame))]): Datasets with data & header
+
+    Example:
+
+    datasets = {
```

Comment:
is this an example of a return?

A user would typically expect an example of code with outputs, if you have an "example" section. Perhaps clarify